src/transformers/generation/configuration_utils.py

```diff
@@ -188,7 +188,15 @@ class GenerationConfig(PushToHubMixin):
         guidance_scale (`float`, *optional*):
             The guidance scale for classifier free guidance (CFG). CFG is enabled by setting `guidance_scale > 1`.
             Higher guidance scale encourages the model to generate samples that are more closely linked to the input
-            prompt, usually at the expense of poorer quality.
+            prompt, usually at the expense of poorer quality. If `negative_prompt` is unset, this will set it to the
```

Comment:
This `negative_prompt` part should be describing the next argument, not the guidance scale!